client/src/components/DataFiles/DataFilesListing/DataFilesListing.jsx

```diff
@@ -168,7 +168,8 @@ const DataFilesListing = ({ api, scheme, system, path, isPublic }) => {
     path
   ) {
     if (isAtHomeDir) {
-      return isRootDir ? 'Root' : systemDisplayName;
+      // For WP-420 search placeholder needs to change when going into a new project, was "Root" before. Needs to show current folder instead
```

Comment:
Good comment for this PR, but does not need to be in code. Instead, you can annotate your PRs by self-reviewing and adding comments to the PR
```suggestion
```

Reply:
Thank you for the tip Sal! I'll go ahead and add that to the PR review right now!